routing/rt/visualizer/config.go

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) 2016 Bitfury Group Limited
+// Distributed under the MIT software license, see the accompanying
+// file LICENSE or http://www.opensource.org/licenses/mit-license.php
+
+package visualizer
```

Comment:
All publicly exported variables in this package are missing godoc-style comments. Also, I think this package should be converted to an internal golang package.